wirelength_analyzer/xcvup_device_data.py

```diff
@@ -53,6 +53,17 @@ def __contains__(self, item):
             'RAMB36E2':        self.default_sequential,
             'RAMB18E2':        self.default_sequential,
 
+            'MMCME4_ADV':      self.default_sequential,
```

Comment:
I think `default_sequential` is no longer appropriate for some of these new blocks. Perhaps we should remain this and `default_combinatorial` to `none_to_none` and `all_to_all` respectively to match the webpage?

Reply:
renamed here: https://github.com/Xilinx/fpga24_routing_contest/pull/40/commits/9c8a3c73adbcc5c7ccd86db82d51d55c7d119182